setup_environment.sh

```diff
@@ -43,7 +43,10 @@ done
 sudo apt-get update
 sudo apt-get install -y python-virtualenv
 
-virtualenv subbots_python
+if [ ! -d $CURR_DIR/subbots_python ]; then
```

Comment:
Ah, a classic moment where bash has *again* confused the heck out of me. What are we checking here? A quick comment would be really helpful.

Reply:
Yeah I'll add a comment, virtualenv throws a warning if the enviroent you are trying to create already exists, this causes the script to exit so I just added a line to check if the virtual environment already exists. I'll add a comment